tests/integration/conftest.py

```diff
@@ -1,11 +1,12 @@
 from collections import namedtuple
 from enum import Enum
 from datetime import datetime
+from labelbox.schema.labeling_frontend import LabelingFrontend
```

Comment:
Bad import order. Let's follow https://www.python.org/dev/peps/pep-0008/#imports. We also use a few other conventions (mostly driven by Greg's PyCharm) in other places, that we should sync on and decide what we want to propagate in the SDK.